### PR TITLE
Editorial suggestions

### PR DESCRIPTION
**Pull request by gurshabad**
Happy to elaborate on them if the motivation is not clear from the changes, let me know!

### CODE REVIEW

File: draft-ietf-capport-architecture.xml [hunk 1]
```diff
@@ -323,21 +326,23 @@
...
         <t>
        The Captive Portal Enforcement component:
         </t>
         <ul spacing="normal">
-          <li>Allows traffic through for allowed User Equipment.</li>
-          <li>Blocks (discards) traffic for disallowed User Equipment.</li>
+          <li>Allows traffic through for allowed User Equipment that has satisfied
+          the Captive Portal Conditions.</li>
+          <li>Blocks (discards) traffic according to the site-specific policy
+            for User Equipment has not yet satisfied the Captive Portal Conditions.</li>
```
Comment (klarose) [inline draft-ietf-capport-architecture.xml]:
```suggestion
            for User Equipment that has not yet satisfied the Captive Portal Conditions.</li>
```

File: draft-ietf-capport-architecture.xml [hunk 2]
```diff
@@ -347,13 +352,13 @@
         <t>
         User Equipment may send traffic outside the captive network prior to the Enforcement
         device granting it access. The Enforcement Device rightly blocks or resets these
-        requests. However, lacking a signal from the Enforcement Device or interaction with the
+        requests. However, in the absence of a signal from the Captive Network or interaction with the
```
Comment (klarose) [inline draft-ietf-capport-architecture.xml]: Is there a particular reason you preferred referring to the Captive Network as opposed to the Enforcement Device? The Enforcement Device is the component within the network responsible for sending the signal. It makes sense to me that it be the thing that has not sent the signal to the User Equipment in this paragraph.

Comment (gurshabad) [reply draft-ietf-capport-architecture.xml]: You're right, of course. I made this edit without reading the entire section (when I was slightly confused, as I noted in my email), and forgot to remove this edit.

509d855e5ddba7dd1c5db58646c85516d0b40987 corrects it.

**Comment by gurshabad** [comment]
Thanks @klarose!

b1dc7ba4e26ab17ed12d8a5a2408fe8063b1177a accepts the suggestion.

40c2d58a6d398243aae79fa6fb8a547c167f3c72 corrects the mistake I had made, which your question was about.

**Pull request merged**